src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with [images](/en/guides/images/), there will be times where you will need to dynamically import the image paths. 
+
+In this recipe, you will learn how to dynamically import your images the correct way using Vite's `import.meta.glob` function. 
```

Comment:
```suggestion
In this recipe, you will learn how to dynamically import your images using Vite's `import.meta.glob` function. You will build a card component that displays the name, age, and photo of a person.
```
Probably don't need to maybe accidentally judge people here! They're just going to learn how to do it via X.  :sweat_smile: 

Also, I like mentioning what we're building up front in this case especially because we're going to define some props right away related to this, but not strictly necessary for importing images to work. I think this gives a little extra helpful context for that!